src/recipes/custom-resources.md

```diff
@@ -0,0 +1,36 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Custom Resources
+
+Custom `Resource`s are exposed for the end user to use within their development, but cannot provide any functionality automatically.
+For example, you can create a custom `AudioStream` type that handles a new and interesting audio file type.
+
+
+## Registering a `Resource`
+
+This workflow is similar to the [Hello World example][hello]:
+
+```rust
+#[derive(GodotClass)]
+#[class(tool, init, base=Resource)]
+struct ResourceType {
+    #[base]
+    base: Base<Resource>,
+}
+```
+
+It is important to note that similar to defining custom resources in GDScript, marking this class as a "tool class"
+is required for expected functionality.
+
+The above resource does not export any variables. While not all `Resource`s will require exported variables, most do.
```

Comment:
Be consistent with using either symbol notation or noun -- the preceding sentence uses "resource", so that's fine here, too. Also, I would omit "will".
```suggestion
The above resource does not export any variables. While not all resources require exported variables, most do.
```